### DISCUSSION TITLE
DataImporter: Use a name other than "import-xxxxx.." for imported data.

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#16650] [Q&A / Support] [answered]
As already mentioned by the user "papiermuecke7626" in the following video https://www.youtube.com/watch?v=5iCtrzL3LUA -  

"Is there a way to import the import-data with a sequential number? Because in the data objects all are called so far only: import-xxxxxxxxxx.xxxxxxxx
e.g. that the data are imported as: file_01, file_02, file_03 ..."

I would like to achieve this too. But until now I had to write a custom console command that renames the data. But I would like to do this directly via the importer

**Comment by Kaltriina** [comment, marked as answer]
You can create new mapping called key with FieldName SYSTEM Key and based on the source attribute you will have the name of the importing

**Reply by ghost** [reply]
Thank you!
